pymeasure/instruments/anritsu/anritsuMS2090A.py

```diff
@@ -0,0 +1,516 @@
+        '''
+        Returns the sync power measurement in json format.
+        '''
+    )
+
+    fet_ocupied_bw = Instrument.measurement(
+        "FET:OBW%g?",
+        '''
+        Returns the different set of measurement information depending on the suffix.
+        ''',
+        validator=truncated_range,
+        values=[1, 2]
+    )
+
+    fet_ota_mapping = Instrument.measurement(
+        "FET:OTA:MAPP?",
+        '''
+        Returns the most recent Coverage Mapping measurement result.
+        '''
+    )
+
+    fet_pan = Instrument.measurement(
+        "FET:PAN?",
+        '''
+        Return the current Pulse Analyzer measurement data. JSON format
+        '''
+    )
+
+    fet_pci = Instrument.measurement(
+        "FET:PCI?",
+        '''
+        Returns PCI measurements
+        '''
+    )
+
+    fet_pdsch = Instrument.measurement(
+        "FET:PDSC?",
+        '''
+        Returns the Data Channel Measurements in JSON format.
+        '''
+    )
+
+    fet_peak = Instrument.measurement(
+        "FET:PEAK?",
+        '''
+        Returns a pair of peak amplitude in current sweep.
+        '''
+    )
+
+    fet_rrm = Instrument.measurement(
+        "FET:RRM?",
+        '''
+        Returns the Radio Resource Management in JSON format.
+        '''
+    )
+
+    fet_scan = Instrument.measurement(
+        "FET:SCAN?",
+        '''
+        Returns the cell scanner measurements in JSON format
+        '''
+    )
+
+    fet_semask = Instrument.measurement(
+        "FET:SEM?",
+        '''
+        This command returns the current Spectral Emission Mask measurement result.
+        '''
+    )
+
+    fet_ssb = Instrument.measurement(
+        "FET:SSB?",
+        '''
+        Returns the SSB measurement
+        '''
+    )
+
+    fet_sync_evm = Instrument.measurement(
+        "FET:SYNC:EVM?",
+        '''
+        Returns the Sync EVM measurement in JSON format.
+        '''
+    )
+
+    fet_sync_power = Instrument.measurement(
+        "FET:SYNC:POW?",
+        '''
+        Returns the sync power measurements in JSON format
+        '''
+    )
+
+    fet_tae = Instrument.measurement(
+        "FET:TAE?",
+        '''
+        Returns the Time Alignment Error in JSON format.
+        '''
+    )
+
+    init_continuous = Instrument.control(
+        "INIT:CONT?", "INIT:CONT %g",
+        "Specified whether the sweep/measurement is triggered continuously",
+        values=ONOFF,
+        map_values=True
+    )
+
+    init_sweep = Instrument.setting(
+        "INIT",
+        '''
+        Initiates a sweep/measurement.
+        '''
+    )
+
+    init_sweep_all = Instrument.setting(
+        "INIT:ALL",
+        '''
+        Initiates sweep until all active traces reach its average count
+        '''
+    )
+
+    init_spa_self = Instrument.measurement(
+        "INIT:SPA:SELF?",
+        docs='''
+        Perform a self-test and return the results.
+        '''
+    )
+
+    inst_active_state = Instrument.control(

```

Comment:
You do not have to imitate the communication while naming properties. Would not "active_state" be better?